source/data-channel.js

```diff
@@ -180,6 +180,40 @@ Skylink.prototype._createDataChannel = function(peerId, dataChannel, bufferThres
   }
 };
 
+/**
+ * Function that refreshes the main messaging Datachannel.
+ * @method refreshDatachannel
+ * @private
+ * @for Skylink
+ * @since 0.6.29
+ */
+Skylink.prototype.refreshDatachannel = function (peerId) {
+
+  var self = this;
+  if(typeof self._dataChannels[peerId] != 'undefined'
+    && typeof self._dataChannels[peerId]["main"] != 'undefined') {
```

Comment:
Perhaps we shouldn't have it `typeof != "undefined"`? Because the value can be `null` and the object doesn't exist in that same way. Also if we know the property `["main"]` exists, `self._dataChannels[peerId].main` would make more sense perhaps?